### ISSUE TITLE
gap parameter?

### ISSUE THREAD

**Issue by dimskor** [gilbitron/flexmasonry#7]
Is it possible to set gap parameter? I tried adding it with css, but it breaks the grid

**Comment by tourdefran** [comment]
@thedimiks Hi, did you find a solution to this? I'm trying to handle the gaps too (both vertical and horizontal)
thanks in advance

**Comment by mapsteps** [comment]
I figured just adding padding to the inner div is doing the trick.

**Comment by tourdefran** [comment]
@MapSteps, thanks for the answer, not so fancy but if it works...

**Comment by christianwalter** [comment]
@MapSteps how did you handle the layout shift with the padding? I need to align the items vertically with my grid.

**Comment by mapsteps** [comment]
@christianwalter,

I added 10px of padding to the inner div and the following to the outer wrapper as - in my case - inline styles:

`style="margin-left: -10px; width: calc(100% + 20px);"`

Hope that helps :)

**Comment by christianwalter** [comment]
@MapSteps thx, that didn't work for me... Just made grid variante with adding the padding for with and max-width, quiet similar to your solution but without a negative margin.

**Comment by mapsteps** [comment]
Hey @christianwalter,

I just realized that I didn't add that to the outer wrapper but instead to another wrapper that's around the entire gallery.
You can actually have a look at what I did here - https://portraitmode.io/.

**Comment by DIntriglia** [comment]
+1 for this requirement as well. Will be running through some tests on my side to see if I can implement a PR for this similar to @cinkagan's above but without the formatting.

@gilbitron Are you willing to merge if I can PR a working copy ideally with tests?

**Comment by gilbitron** [comment]
@DIntriglia Sure happy to merge a working PR with tests 👍

**Comment by TomasBalbinder** [comment]
Hello, I don't know how to create gaps anymore? I need to have them lined up. Thank you so much

**Comment by briannehinchliffe** [comment]
@cinkagan, @DIntriglia any update on this? This library is essentially perfect besides the lack of a gap parameter and it's sad to know that a fix essentially exists, we're just waiting on a PR.

Thanks!

**Comment by DIntriglia** [comment]
@briannehinchliffe I apologize. I forgot to follow up and actually forget the original reason of me using this library. I will need to look in my projects and see if I can find how I solved the issue.

**Comment by DIntriglia** [comment]
Just created a new PR based on #10 – This should resolve the issue while keeping the project clean.

**Comment by briannehinchliffe** [comment]
@DIntriglia no need to apologize! I appreciate you taking a look at this again!

**Comment by briannehinchliffe** [comment]
@gilbitron Would love if you had a minute to review #18  with @DIntriglia's great work! Thanks so much 😄
